test/index.test.ts

```diff
@@ -1,23 +1,18 @@
 import test from 'ava'
-import {
-  Entity,
-  Transform,
-  Vector3,
-  BoxShape,
-  Quaternion,
-  GLTFShape,
-  SphereShape
-} from 'decentraland-ecs'
+import * as DCL from 'decentraland-ecs'
 
 import SceneWriter from '../src'
 
 test('Unit - write entity cube - should output TS of cube', t => {
-  const sceneWriter = new SceneWriter()
-  sceneWriter.addEntity('cube', new Entity())
-  sceneWriter.addComponent('cube', new BoxShape())
+  const sceneWriter = new SceneWriter(DCL)
+  sceneWriter.addEntity('cube', new DCL.Entity())
+  sceneWriter.addComponent('cube', new DCL.BoxShape())
   sceneWriter.addComponent(
     'cube',
-    new Transform({ position: new Vector3(5, 0, 5), rotation: new Quaternion(0, 0, 1, 0) })
+    new DCL.Transform({
+      position: new DCL.Vector3(5, 0, 5),
+      rotation: new DCL.Quaternion(0, 0, 1, 0)
+    })
   )
```

Comment:
Going back to the comment of the components. After that change, this test will look like this:
```ts
  const sceneWriter = new SceneWriter(DCL)
  const cubeEntity = new DCL.Entity()
  cubeEntity.set(new DCL.BoxShape())
  cubeEntity.set(
    new DCL.Transform({
      position: new DCL.Vector3(5, 0, 5),
      rotation: new DCL.Quaternion(0, 0, 1, 0)
    })
  )
  sceneWriter.addEntity('cube', cubeEntity)
```

Which also adds the value of the component validations in `DCL.Entity#set`